### ISSUE TITLE
Delete button for firewall Egress rules located in wrong column

### ISSUE THREAD

**Issue by kricud** [apache/cloudstack#5268]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.15.1
~~~

##### SUMMARY

Delete button for firewall Egress rules located in wrong column

##### STEPS TO REPRODUCE

![Screenshot 2021-08-02 164635](https://user-images.githubusercontent.com/30048481/127871863-c7b2fde1-a3ae-432b-beb0-3d703c41d872.png)
~~~
Create and delete egress firewall rules
~~~

##### EXPECTED RESULTS

~~~
Delete button located in Action column
~~~

##### ACTUAL RESULTS

~~~
Delete button located in ICMP Code / End Port column
~~~

**Comment by shwstppr** [comment]
@kricud what browser are you seeing this? I'm not able to reproduce this with 4.15.1.0 on Google Chrome and Firefox,
![Screenshot from 2021-08-03 10-54-13](https://user-images.githubusercontent.com/153340/127962435-cc1db6fa-f255-4a06-adbf-1d4785e26267.png)

**Comment by kricud** [comment]
I can repeat this on Chrome Version 92.0.4515.107 (Official Build) (64-bit), Firefox Version 90.0.2 (64-bit) and Edge Version 92.0.902.62 (Official build) (64-bit).

@shwstppr you will reproduce problem by creating rule as in screenshot below:

![Screenshot 2021-08-03 084919](https://user-images.githubusercontent.com/30048481/127964691-5512584c-a102-4f29-bab9-72a5fa159645.png)

**Comment by kricud** [comment]
@shwstppr tested in production without any customizations raw 4.15.1.0 still can repeat it.

![Screenshot 2021-08-03 091103](https://user-images.githubusercontent.com/30048481/127966673-ba13c8bd-59c2-4e65-b3f7-38014212f5ca.png)

**Comment by shwstppr** [comment]
@kricud create PR #5269  to fix the issue. Please test if possible

**Comment by kricud** [comment]
@shwstppr vacation time don’t have capability to build it. +2 weeks to test it. Fix #5269 LGTM.
Close it and I will test it from nightly and reopen if something.

**Comment by nvazquez** [comment]
Closing as PR #5269 is merged
